site/en/tutorials/keras/save_and_load.ipynb

```diff
@@ -798,14 +798,14 @@
         "id": "dGXqd4wWJl8O"
       },
       "source": [
-        "Keras saves models by inspecting the architecture. This technique saves everything:\n",
```

Comment:
What do you all think about changing "architecture" to plural? -> "Keras saves models by inspecting their architectures." (models<->their<->architectures)

Reply:
sounds slightly better to me. done and done.